.github/workflows/test_conan_extensions.yml

```
@@ -18,7 +21,14 @@ jobs:
     - name: Install dependencies
       run: |
         pip install -U pip
-        pip install pytest git+https://github.com/conan-io/conan.git@release/2.0
-    - name: Conan latest - Python 3.6
+        pip install pytest
+    - name: Install Conan
+      run: |
+        if [ "${{ matrix.conan-version }}" == "latest" ]; then
+          pip install conan
+        elif [ "${{ matrix.conan-version }}" == "develop" ]; then
+          pip install git+https://github.com/conan-io/conan.git@release/2.0
+        fi
```

What about something like this? Note that instead of develop we could use the branch name and have an easier time updating when we eventually move to release/2.1 

(Syntax not checked, sent from phone)

```suggestion
    - name: Install Latest Conan
      if: matrix.conan-version == "latest"
      run: pip install conan
   - name: Install ${{ matrix.conan-version }} 
     if: matrix.conan-version != "latest"
     run: pip install git+https://github.com/conan-io/conan.git@${{ matrix.conan-version }}
```

---

this approach was not working as tasks were run in parallel and caused issues with artifactory